pkgs/build-support/fetchurl/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, curl }: # Note that `curl' may be `null', in case of the native stdenv.
+{ stdenv, curl, ca_cert_file ? null, insecure ? false }: # Note that `curl' may be `null', in case of the native stdenv.
```

Comment:
I want to make sure all implementations explicitly understand that a lack
of ca implies insecure but I wouldn't be against changing this.

On Sun, May 31, 2015, 01:53 Vladimír Čunát <notifications@github.com> wrote:

> In pkgs/build-support/fetchurl/default.nix
> <https://github.com/NixOS/nixpkgs/pull/8082#discussion_r31388309>:
>
> > @@ -1,4 +1,4 @@
> > -{ stdenv, curl }: # Note that `curl' may be `null', in case of the native stdenv.
> > +{ stdenv, curl, ca_cert_file ? null, insecure ? false }: # Note that `curl' may be `null', in case of the native stdenv.
>
> What about dropping the insecure parameter and consider let insecure =
> ca_cert_file != null;? Is there any sense in a different setting?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/8082/files#r31388309>.
>